pkg/util/github/github.go

```diff
@@ -125,7 +125,7 @@ func DownloadOpenShiftReleaseBinary(binaryType OpenShiftBinaryType, osType minis
 		return errors.Wrap(err, fmt.Sprintf("Cannot download OpenShift release asset %d", assetID))
 	}
 	if len(url) > 0 {
-		glog.V(2).Infof("Downloading %s %s\n", binaryType.String(), *release.TagName)
+		fmt.Println(fmt.Sprintf("Downloading OpenShift client binary %s with version '%s'", binaryType.String(), *release.TagName))
```

Comment:
@budhrg Sure. Thats better I think. I had missed the ` *release.TagName` in @hferentschik suggestion.